.github/workflows/http.yml

```diff
@@ -53,11 +53,12 @@ jobs:
     steps:
       - name: Setup
         run: |
+          SANITIZED_TAG=$(echo "${GITHUB_REF_NAME}" | sed 's/[./]/-/g')
           echo "EXEC_NAME=albyhub" >> $GITHUB_ENV
           if [ "${{ runner.os }}" == "macOS" ]; then
-            echo "PACKAGE_NAME=albyhub-Server-MacOS" >> $GITHUB_ENV
+            echo "PACKAGE_NAME=AlbyHub-Server-MacOS-${SANITIZED_TAG}" >> $GITHUB_ENV
           else
-            echo "PACKAGE_NAME=albyhub-Server-Linux-${{ matrix.build.arch }}" >> $GITHUB_ENV
+            echo "PACKAGE_NAME=AlbyHub-Server-Linux-${{ matrix.build.arch }}-${SANITIZED_TAG}" >> $GITHUB_ENV
```

Comment:
will this work with our server update script?

Reply:
I actually think file names should always be lowercased. If we change it then let's make sure everything is lowercase, then we less likely have issues on case-sensitive systems.

Reply:
Cool, changing it to be lowercase 👍

Reply:
I'm not sure this is the only issue. I think the Alby download links are to exact filenames, but now we add a version suffix on the filename. I think a change in getalby.com is also needed to ensure it picks up the file.

I originally suggested this because downloading two versions with the same filename ends up suffixing the second one with a (1).

Reply:
> I originally suggested this because downloading two versions with the same filename ends up suffixing the second one with a (1).

that's the reason why we add the version number to the filename? why would a user do that and even if the user does that why would it be a a problem?